spec/person_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+describe Person do
+  before :each do
+    @person = Person.new(22, 'Name', parent_permission: true)
+    @book = Book.new('Title', 'Author')
+  end
+
+  describe '#new' do
+    it 'takes three parameters and returns a Person instance' do
+      expect(@person).to be_an_instance_of(Person)
+    end
+  end
+
+  describe '#correct_name' do
+    it 'Return the name of the person' do
+      expect(@person.correct_name).to eq('Name')
+    end
+  end
+
+  describe '#can_use_services' do
+    it 'Return true if person is older than 18 or have permission, expect true "current age 22"' do
+      expect(@person.can_use_services?).to eql(true)
+    end
+  end
+
```

Comment:
- [ ] It would be better if we test for the method `is_of_age` in our person class